default/python/AI/FreeOrionAI.py

```diff
@@ -313,7 +313,7 @@ def generateOrders():  # pylint: disable=invalid-name
     # TODO: Consider adding an option to clear AI orders after load (must save AIstate at turn start then)
     if fo.currentTurn() == foAIstate.last_turn_played:
         info("The AIstate indicates that this turn was already played.")
-        if not check_bool(get_option_dict().get('replay_turn_after_load', 'False')):
+        if not check_bool(get_option_dict().get('replay_turn_after_load', 'False')) and fo.currentTurn() > 1:
```

Comment:
Not sure what review you want from me, but `if (not((something is false) and (something else is true))` is bit hard to parse. Maybe `if ((something is true) or (something else is false))` could be a bit easier?

Reply:
if you think it's safe, and since we probably need an RC2 anyway, sure...